### ISSUE TITLE
Cannot reproduce validation accuracy

### ISSUE THREAD

**Issue by marsupialtail** [liuyang079/vgg-tensorflow-cifar10#1]
Hello!

I am trying to run the repo as is with the CIFAR dataset. Somehow, the model severely overfits, i.e. 0 training error but 5 validation loss. Have you encountered similar issues? I am training the model from scratch. 

Thank you!

Ziheng Wang
